src/tlo/methods/healthsystem.py

```diff
@@ -830,6 +830,11 @@ def pre_initialise_population(self):
         self.tclose_overwrite = self.parameters['tclose_overwrite']
         self.tclose_days_offset_overwrite = self.parameters['tclose_days_offset_overwrite']
 
+        # Ensure name of policy we want to consider before/after switch is among the policies loaded
+        # in the self.parameters['priority_rank']
+        assert self.parameters['policy_name'] in self.parameters['priority_rank'].keys()
+        assert self.parameters['policy_name_post_switch'] in self.parameters['priority_rank'].keys()
```

Comment:
You don't need `.keys()` e.g. `assert self.parameters['policy_name'] in self.parameters['priority_rank']` is sufficient